### PR TITLE
(feat): add openebs pool container failure experiment CR

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  modify generic experiments to add lib_image env

- add openebs pool container failure experiment CR

### CODE REVIEW

**Review by chandankumar4 — DISMISSED** [review]
LGTM

**Review by chandankumar4 — DISMISSED** [review]

File: charts/openebs/openebs-pool-container-failure/openebs-pool-container-failure.chartserviceversion.yaml [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-  displayName: openebs-pool-failure 
+  displayName: openebs-pool-container-failure 
   categoryDescription: >
      Kill the pool pod and check if gets scheduled again. 
   keywords:
     - Kubernetes
     - Storage Pool
     - OpenEBS
+  Platforms:
+    - Gke
```
Comment (ksatchit) [inline charts/openebs/openebs-pool-container-failure/openebs-pool-container-failure.chartserviceversion.yaml]: GKE

**Review by chandankumar4 — DISMISSED** [review]

**Comment by ksatchit** [comment]
@shubhamchaudhary can you pls resolve conflicts

**Review by chandankumar4 — DISMISSED** [review]

File: charts/openebs/openebs-pool-container-failure/openebs-pool-container-failure.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  version: 0.1.1
+  annotations:
+    categories: OpenEBS
+    vendor: CNCF
+    repository: https://github.com/litmuschaos/chaos-charts
+    support: https://slack.openebs.io/
+spec:
+  displayName: openebs-pool-container-failure 
+  categoryDescription: >
+     Kill the pool pod and check if gets scheduled again.
```
Comment (Sanjay1611) [inline charts/openebs/openebs-pool-container-failure/openebs-pool-container-failure.chartserviceversion.yaml]: It should be pool container.

File: charts/generic/pod-network-latency/experiment.yaml [hunk 3]
```diff
@@ -34,24 +34,26 @@ spec:
...
 
     - name: NETWORK_INTERFACE
       value: 'eth0'
 
-    - name: NETWORK_LATENCY
+    # provide lib image
+    - name: LIB_IMAGE
+      value: 'gaiaadm/pumba:0.4.8' 
+
+   - name: NETWORK_LATENCY
```
Comment (Sanjay1611) [inline charts/generic/pod-network-latency/experiment.yaml]: Indentation is not right

File: charts/generic/pod-delete/experiment.yaml [hunk 4]
```diff
@@ -40,11 +40,14 @@ spec:
...
+    - name: FORCE
+      value: 'true'
+
     - name: CHAOS_INTERVAL
       value: '5'
 
     - name: LIB
       value: ''
-
+
```
Comment (Sanjay1611) [inline charts/generic/pod-delete/experiment.yaml]: no space should be there

File: charts/openebs/openebs-pool-container-failure/experiment.yaml [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Kill the pool pod and check if gets scheduled again
```
Comment (Sanjay1611) [inline charts/openebs/openebs-pool-container-failure/experiment.yaml]: pool `container`

**Review by Sanjay1611 — CHANGES_REQUESTED** [review]

**Review by chandankumar4 — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**